.version

```diff
@@ -1 +1 @@
-0.6.0
+0.7.0
```

Comment:
Heads up as you are resolving the conflicts, my recent PR (#95) updated the version to 0.7.0, so you may want to go to 0.8.0 here. Similarly, I updated the helm chart to version 1.4.22, so probably update that to 1.4.23.

Reply:
Oh, and my change updated the schema version to 0.4.0, so it would probably be wise to update the schema version to 0.5.0 as well.

Reply:
Looks good to me.